components/playlist/browser/resources/reducers/states.ts

```diff
@@ -49,3 +60,9 @@ export const useTotalDuration = (playlist?: Playlist) => {
     }, 0)
   }, [playlist])
 }
+
+export function useTotalSize (playlist?: Playlist) {
+  return React.useMemo(() => {
+    return playlist?.items ? getFormattedTotalBytes(playlist.items) : ''
```

Comment:
I don't know if it's desirable, but this would display `0mb`, which might be nice?
```suggestion
    return getFormattedTotalBytes(playlist?.items ?? [])
```